examples/index.json

```diff
@@ -444,22 +444,22 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Converting from user-defined type to JSON",
-                "url": "converting-from-user-defined-type-to-json",
+                "name": "Convert from user-defined type to JSON",
+                "url": "convert-from-user-defined-type-to-json",
                 "verifyBuild": true,
                 "verifyOutput": true,
                 "isLearnByExample": true
             },
             {
-                "name": "Converting from JSON to user-defined type",
-                "url": "converting-from-json-to-user-defined-type",
+                "name": "Convert from JSON to user-defined type",
+                "url": "convert-from-json-to-user-defined-type",
```

Comment:
These will result in quite a few broken links, right? I'm not sure if that's acceptable given the number of BBEs we're changing.

Reply:
Reverted the URL changes.